Miniscript-Template-001-3-Key-Time-Layered-Multisig.mediawiki

```diff
@@ -0,0 +1,63 @@
+Miniscript Template: 1
+Title: 3 Key Time Layered Multisig
+Created: 2023-09-11
+
+=Name of Template=
+
+3 Key Degrading Multisig
+
+=Goal to be achieved by template=
```

Comment:
In e2167a422b8504dad625b538d3b3c39d55e39640:

Similarly here, the "Name of Template" section should be dropped and the "Goal to be Achieved" section should be renamed "Motivation".